site/kubernetes/operator/using-topology-operator.md

```diff
@@ -0,0 +1,231 @@
+
+<pre class="lang-bash">
+apiVersion: rabbitmq.com/v1beta1
+kind: Queue
+metadata:
+  name: test # name of this custom resource; does not have to the same as the actual queue name
+  namespace: rabbitmq-system
+spec:
+  name: test # name of the queue
+  autoDelete: false
+  durable: true
+  rabbitmqClusterReference:
+    name: example-rabbit
+</pre>
+
+The following manifest will create a policy named 'lazy-queue' in default vhost:
+
+<pre class="lang-bash">
+apiVersion: rabbitmq.com/v1beta1
+kind: Policy
+metadata:
+  name: policy-example # name of this custom resource
+  namespace: rabbitmq-system
+spec:
+  name: lazy-queue
+  pattern: "^lazy-queue-" # matches any queue begins with "lazy-queue-"
+  applyTo: "queues"
+  definition:
+    queue-mode: lazy
+  rabbitmqClusterReference:
+    name: example-rabbit
+</pre>
+
+The Messaging Topology repo has more examples on [queues](https://github.com/rabbitmq/messaging-topology-operator/tree/main/docs/examples/queues)
+and [policies](https://github.com/rabbitmq/messaging-topology-operator/tree/main/docs/examples/policies). 
+
+### <a id='binding' class='anchor' href='#binding'> Exchanges and bindings</a>
+
+You can use Messaging Topology Operator to create [exchanges](https://www.rabbitmq.com/publishers.html#basics) and [bindings](https://www.rabbitmq.com/e2e.html).
+The following manifest will create a fanout exchange:
+
+<pre class="lang-bash">
+apiVersion: rabbitmq.com/v1beta1
+kind: Exchange
+metadata:
+  name: fanout
+  namespace: rabbitmq-system
+spec:
+  name: fanout-exchange # name of the exchange
+  type: fanout # default to 'direct' if not provided; can be set to 'direct', 'fanout', 'headers', and 'topic'
+  autoDelete: false
+  durable: true
+  rabbitmqClusterReference:
+    name: example-rabbit
+</pre>
+
+The following manifest will create a binding between an exchange and a queue:
+
+<pre class="lang-bash">
+apiVersion: rabbitmq.com/v1beta1
+kind: Binding
+metadata:
+  name: binding
+  namespace: rabbitmq-system
+spec:
+  source: test # an existing exchange
+  destination: test # an existing queue
+  destinationType: queue # can be 'queue' or 'exchange'
+  rabbitmqClusterReference:
+    name: example-rabbit
+</pre>
+
+More examples on [exchanges](https://github.com/rabbitmq/messaging-topology-operator/tree/main/docs/examples/exchanges)
+and [bindings](https://github.com/rabbitmq/messaging-topology-operator/tree/main/docs/examples/bindings). 
+
+### <a id='users' class='anchor' href='#users'>Users</a>
+
+You can use Messaging Topology Operator to create RabbitMQ users and assign user permissions.
+Learn more about user management in RabbitMQ [here](https://www.rabbitmq.com/access-control.html#user-management).
+
+Messaging Topology Operator supports creating users with generated credentials and with provided credentials.
+When creating a user with provided username and password, create a kubernetes secret object contains keys `username` and `password` in its Data field.
```

Comment:
Maybe worth being a bit more explicit about the fact that the operator generates the credentials by default, rather than saying it's supported.